exercises/views/problem.md

```diff
@@ -47,7 +47,7 @@ surrounding the variable with curly braces, e.g. `{{foo}}`.
 
 The template receives some information from the request. For example, the query
 parameters that were passed in via the URL are available in the `query` object.
-These parameters can then be used in the template.
+These parameters can then be used in the template. Query params don't have to be declared in the route `path`.
```

Comment:
If that seems fine feel free to change it (or counter propose and I'll get this merged in).